script.py

```diff
@@ -0,0 +1,31 @@
+import requests
+import pprint
+import login
```

Comment:
a bit nitpicky, but it would be more clear if this file was named config as that is a more standard name for a file containing credentials